### PR TITLE
[CSL-1068] Use new Plutus

### PR DESCRIPTION
**Pull request by neongreen**
This one should actually work probably.

### CODE REVIEW

**Comment by gromakovsky** [comment]
Can we close #1206?

**Comment by neongreen** [comment]
@gromakovsky done.

File: txp/Pos/Txp/Toil/Utxo/Functions.hs [hunk 1]
```diff
@@ -210,17 +209,16 @@ verifyInputs VTxContext {..} resolvedInputs TxAux {..} =
         UnknownWitnessType t _ -> checkUnknownAddressType t addr
 
     -- the second argument here includes local context, can be used for scripts
```
Comment (gromakovsky) [inline txp/Pos/Txp/Toil/Utxo/Functions.hs]: It's the first argument now.

Comment (neongreen) [inline txp/Pos/Txp/Toil/Utxo/Functions.hs]: Okay, I actualized it.

**Pull request merged**